lib/src/utils.dart

```diff
@@ -34,6 +36,9 @@ bool isIdentifier(String name) => _identifier.hasMatch(name);
 /// Returns `true` of the given [name] is composed only of `_`s.
 bool isJustUnderscores(String name) => _underscores.hasMatch(name);
 
+/// Returns `true` if the given [name] has leading `_`s.
+bool hasLeadingUnderscores(String name) => _leadingUnderscores.hasMatch(name);
```

Comment:
It's only the first underscore that matters, so it would be more efficient to test `name.startsWith('_')` than to use a regular expression.

Reply:
Removed regex and updated code.